commands/createfixpullrequests.go

```diff
@@ -184,7 +184,7 @@ func (cfp *CreateFixPullRequestsCmd) openFixingPullRequest(impactedPackage, fixB
 	}
 
 	log.Info("Pushing fix branch:", fixBranchName, "...")
-	err = cfp.gitManager.Push()
+	err = cfp.gitManager.Push(fixBranchName)
 	if err != nil {
 		return
 	}
```

Comment:
Can be simplified to - 
```
	if err = cfp.gitManager.Push(fixBranchName); err != nil {
		return
	}
```
There's another place inside this function to implement the same simplification.